### PR TITLE
fix(TBD-13175): EMR 6.2 fails when job contains tDatamasking component

### PR DESCRIPTION
**Pull request by sponomarova**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.emr62x/plugin.xml [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+        
+        <libraryNeeded context="plugin:org.talend.hadoop.distribution.emr62x"
              id="spark-core_2.12-3.0.1.jar" name="spark-core_2.12-3.0.1.jar"
              mvn_uri="mvn:org.apache.spark/spark-core_2.12/3.0.1"/>
 
          <libraryNeeded context="plugin:org.talend.hadoop.distribution.emr62x"
              id="spark-hive_2.12-3.0.1.jar" name="spark-hive_2.12-3.0.1.jar"
              mvn_uri="mvn:org.apache.spark/spark-hive_2.12/3.0.1"/>
-
+F
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.emr62x/plugin.xml:13]: typo

**Review by AlixMetivier — CHANGES_REQUESTED** [review]

File: main/plugins/org.talend.hadoop.distribution.emr62x/plugin.xml [hunk 2]
```diff
@@ -312,8 +316,8 @@
           mvn_uri="mvn:org.apache.hive/hive-common/3.1.2-amzn-3"/>
           
         <libraryNeeded context="plugin:org.talend.hadoop.distribution.emr62x"
-          id="hive-exec-3.1.2-tlnd-0.jar" name="hive-exec-3.1.2-tlnd-0.jar"
-          mvn_uri="mvn:org.talend.libraries/hive-exec/3.1.2-tlnd-0"/>
+          id="hive-exec-3.1.2.jar" name="hive-exec-3.1.2.jar"
+          mvn_uri="mvn:org.apache.hive/hive-exec/3.1.2"/>
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.emr62x/plugin.xml:320]: changing this does not lead to issues with the commons-lang3 that is inside default hive exec ?

Comment (cbadillo1603) [reply main/plugins/org.talend.hadoop.distribution.emr62x/plugin.xml:320]: Indeed, this tlnd-0 jar was created to remove the commons-lang3 as it was throwing issues in some jobs.

Comment (lbourgeois) [reply main/plugins/org.talend.hadoop.distribution.emr62x/plugin.xml]: Indeed this one brings the conflict on commons-lang3. I tried to use the -amzn-2 from cluster but it has other issues. Finlly I figured out that we have to use the the [3.1.2-core](https://repo1.maven.org/maven2/org/apache/hive/hive-exec/3.1.2/hive-exec-3.1.2-core.jar) which does not shade anything

**Review by cbadillo1603 — DISMISSED** [review]
reviewed

File: main/plugins/org.talend.hadoop.distribution.emr62x/plugin.xml [hunk 3]
```diff
@@ -308,28 +312,24 @@
...
         <libraryNeeded context="plugin:org.talend.hadoop.distribution.emr62x"
-          id="hive-jdbc-3.1.2-amzn-3.jar" name="hive-jdbc-3.1.2-amzn-3.jar"
-          mvn_uri="mvn:org.apache.hive/hive-jdbc/3.1.2-amzn-3"/>
+          id="hive-jdbc-3.1.2.jar" name="hive-jdbc-3.1.2.jar"
+          mvn_uri="mvn:org.apache.hive/hive-jdbc/3.1.2"/>
 
         <libraryNeeded context="plugin:org.talend.hadoop.distribution.emr62x"
-          id="hive-llap-common-3.1.2-amzn-3.jar" name="hive-llap-common-3.1.2-amzn-3.jar"
-          mvn_uri="mvn:org.apache.hive/hive-llap-common/3.1.2-amzn-3"/>
+          id="hive-llap-common-2.3.7.jar" name="hive-llap-common-2.3.7.jar"
```
Comment (cbadillo1603) [inline main/plugins/org.talend.hadoop.distribution.emr62x/plugin.xml:327]: I approved, but also got a doubt about the version of this jar. Could you confirm this is the expected one.

Comment (lbourgeois) [reply main/plugins/org.talend.hadoop.distribution.emr62x/plugin.xml]: No it isn't. Forgot to update it

**Review by cbadillo1603 — APPROVED** [review]
reviewed

**Review by AlixMetivier — APPROVED** [review]

**Pull request merged**